lib.js

```diff
@@ -48,13 +63,15 @@ var re_all = new RegExp("\\b(" + re_number.source + ")\\s?("
 */
 function convertToMetric(text, callback) {
   callback = callback || (x => x)
-  return text.replace(re_all, (oldValue, value, _, unit) => {
+  return text.replace(re_all, (oldText, value, _, unit) => {
     let conv = conversions.find(c => unit.match(c.regex))
+    // some people might use char code 8722 instead of 45...
```

Comment:
At least these guys do: https://www.mathsisfun.com/temperature-conversion.html

Reply:
its probably ok for now for the 3 users that ametrican has